linting/code-linting-eslint.md

```diff
@@ -0,0 +1,143 @@
+
+# Code liniting + ESLint Instructions
+
+### Projected Time
+10-15 minutes
+
+- Lesson: 5-7 min
+- Guided Practice: 5 min (Mock Project)
+- Independed Practice: (Implement it with your projects)
+- Check for Understanding: 1 min
+
+### Prerquisites
+
+Here are topics you must be comfortable with to get the maximum benefit out of this topic:
+
+-  [JavaScript](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-1.md)
```

Comment:
The prereqs should be:
- [Javascript 6 - Object Literals](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
- [VSCode](https://github.com/Techtonica/curriculum/blob/master/vs-code/vscode.md)

Please note all links to other curriculum materials must be relative links.

Reply:
I'll be right on it. The links provided are include:
- YouTube tutorial for setting up eslint
- Official ESLint Documentation
- Official links to ESLint extension(VSCode, Brackets, Sublime text 3 etc.)